spec/save_operation_callbacks_spec.cr

```diff
@@ -64,6 +68,7 @@ describe "Avram::SaveOperation callbacks" do
   it "runs all callbacks when saving successfully" do
     post = PostBox.create
     operation = CallbacksSaveOperation.new(post)
+    operation.set_title_from_param("A Title")
```

Comment:
I think this was being done by setup_required_attributes previously. If you add that back in then I think you won’t need this. 

Or maybe create another operation if you need one with and one without the setup required attributes

Reply:
I had to remove that to test the required. I needed a way for it to fail when that value isn't added. I could create an option to pass in I guess.